doc/source/examples/index.rst

```diff
@@ -12,14 +12,12 @@ an embedded instance or remote session.
    .. grid-item-card:: Embedding Examples
       :link: https://embedding.examples.mechanical.docs.pyansys.com/index.html
       :text-align: center
-      :margin: 2 2 0 0
```

Comment:
if you are removing the margin, i would recommend to add gutter for the cards. we follow 2 2 3 4 pattern as i wrote in some of the comments below.

Reply:
Reverting back to margin